### ISSUE TITLE
Fatal error: Could not find bundle PhoneNumberKit_PhoneNumberKit

### ISSUE THREAD

**Issue by luisafcastano** [marmelroy/PhoneNumberKit#526]
I am using PhoneNumberKit thought `carthage`. My `Cartfile` has:
```
github "marmelroy/PhoneNumberKit" == 3.3.6
```
When I build my `carthage` dependencies, I see:
```
Carthage
   Checkouts
         PhoneNumberKit
   Build
         iOS/PhoneNumberKit.framework
         iOS/PhoneNumberKit.framework.dSYM
```

When initializing the framework:
```
import PhoneNumberKit

class AppDelegate {
     private lazy var phoneNumberKit = PhoneNumberKit()
}
```

I get this error:
```
Thread 1: Fatal error: Could not find bundle PhoneNumberKit_PhoneNumberKit, tried candidates: 
[
Optional(file:///Users/admin/Library/Developer/CoreSimulator/Devices/E634677F-CA24-41B0-B78F-26231C74EC83/data/Containers/Bundle/Application/5EE366B0-5353-4D1F-B374-E46C3DF5D5B5/kphone.app/), 
Optional(file:///Users/admin/Library/Developer/CoreSimulator/Devices/E634677F-CA24-41B0-B78F-26231C74EC83/data/Containers/Bundle/Application/5EE366B0-5353-4D1F-B374-E46C3DF5D5B5/kphone.app/Frameworks/PhoneNumberKit.framework/), 
```
The file that throws this error is the [Bundle+Resources.swift](https://github.com/marmelroy/PhoneNumberKit/blob/master/PhoneNumberKit/Bundle%2BResources.swift) one.

Any idea how I can get around this?

**Comment by bguidolim** [comment]
This issue was fixed already. Just grab the latest version.

**Comment by bguidolim** [comment]
The latest version is 3.4.0. Or you can use the version with the fix, which is 3.3.7.
